### DISCUSSION TITLE
Block all suggestions from a given domain

### DISCUSSION THREAD

**Discussion by damiensawyer** [dailydotdev/daily#591] [Feature Requests]
Hi, 
I've noticed that the quality of content on a certain blog aggregation site has recently dropped through the floor. Is there a way to block all suggestions for articles on a site?

Thanks

**Comment by sshanzel** [comment]
Hi @damiensawyer! Kindly refer to the link below on how you will be able to manage your sources:
https://docs.daily.dev/docs/settingyourfeed/blocking-tags-sources#blocking-sources

If you want to dive deeper on how to personalize your feed, kindly check this one: https://docs.daily.dev/docs/settingyourfeed/filtering-content-feed

**Reply by damiensawyer** [reply]
Thanks @sshanzel  that's great.
